_sass/_team.sass

```diff
@@ -8,4 +8,9 @@
     @extend .team-avatar
     background-color: rgba(200, 200, 200, 0.9)
     width: 100px
-    height: 100px
\ No newline at end of file
+    height: 100px
+
+a.fa-globe:hover
```

Comment:
this affects more than just team stuff, so i would either:

1. move this to the main css, and make it work for all font awesome icons
2. be more specific and apply it to only the team area

I'm leaning more on the second option as it eliminates any unwanted side effects. at a later stage if we come across more font awesome icons that need this specific change then we can move it accordingly